### ISSUE TITLE
Excel reader Skipping Empty Columns

### ISSUE THREAD

**Issue by IvanAtanasov** [j-easy/easy-batch#319]
The MsExcelRecordReader seems to be skipping empty columns, which to me is not the correct behaviour.

e.g.

Given an excel file like this

| col1 	| col2 	| col3 	|
|------	|------	|------	|
| 1    	|           	| 3    	|
| 1    	| 2    	| 3    	|

When reading using code like this..

```java
Job job = new JobBuilder()
    .named("test job")
    .reader(new MsExcelRecordReader(new File("/path-to-file.xlsx")))
    .mapper(new MsExcelRecordMapper<>(MyPojo.class, "col1", "col2", "col3"))
    .filter(new HeaderRecordFilter())
    .writer(new StandardOutputRecordWriter())
    .build();
```

For the first data row the pojo will have col2 field set as the value from the third column (3) as the second column has been skipped.

Is this expected behaviour?

I'm using version 5.2.0

Thanks,
Ivan

**Comment by benas** [comment]
That's a bug! Thank you for reporting this out 👍 

Let me plan that for next release. Contributions are welcome!

Kr,
Mahmoud

**Comment by benas** [comment]
Fixed with #326 .
